x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/30_date_histogram.yml

```diff
@@ -219,3 +219,30 @@ setup:
   - match: { aggregations.date_histogram.buckets.0.key_as_string: "2021-04-28T18:00:00.000Z" }
   - match: { aggregations.date_histogram.buckets.0.key: 1619632800000 }
   - match: { aggregations.date_histogram.buckets.0.doc_count: 4 }
+
+  # date histogram aggregation on both indices resulting in partial results and shard failures
+  - do:
+      search:
+        index: test,rollup-test
+        body:
+          size: 0
+          aggs:
+            date_histogram:
+              date_histogram:
+                field: "@timestamp"
+                calendar_interval: hour
+                min_doc_count: 1
+
+  - match: { _shards.total: 2 }
```

Comment:
NOTE: here we rely on the fact that the rollup index is created with the same number of shards of the original index, which is set to 1 in the setup block.